aci/resource_aci_vzsubj.go

```diff
@@ -25,6 +27,114 @@ func resourceAciContractSubject() *schema.Resource {
 
 		SchemaVersion: 1,
 
+		CustomizeDiff: func(_ context.Context, diff *schema.ResourceDiff, v interface{}) error {
+			result := false
+			if diff.HasChange("consumer_to_provider") {
+				old, new := diff.GetChange("consumer_to_provider")
+				oldInTerm := old.(*schema.Set).List()
+				newInTerm := new.(*schema.Set).List()
+				if len(newInTerm) == len(oldInTerm) {
+					oldInTermParam := oldInTerm[0].(map[string]interface{})
+					newInTermParam := newInTerm[0].(map[string]interface{})
+
+					if oldInTermParam["prio"] != newInTermParam["prio"] || oldInTermParam["target_dscp"] != newInTermParam["target_dscp"] || oldInTermParam["relation_vz_rs_in_term_graph_att"] != newInTermParam["relation_vz_rs_in_term_graph_att"] {
+						result = true
+					} else if oldInTermParam["relation_vz_rs_filt_att"] != newInTermParam["relation_vz_rs_filt_att"] {
+
+						oldInTermFilterList := oldInTermParam["relation_vz_rs_filt_att"].(*schema.Set).List()
+						newInTermFilterList := newInTermParam["relation_vz_rs_filt_att"].(*schema.Set).List()
+
+						if len(oldInTermFilterList) != len(newInTermFilterList) {
+							result = true
+						} else {
+							for _, oldInTermFilter := range oldInTermFilterList {
+								oldVzRsFiltAttMap := oldInTermFilter.(map[string]interface{})
+								oldInTermFilterDirectives := oldVzRsFiltAttMap["directives"].(*schema.Set).List()
+								found_same_filter_dn := false
+								for _, newInTermFilter := range newInTermFilterList {
+									newVzRsFiltAttMap := newInTermFilter.(map[string]interface{})
+									newInTermFilterDirectives := newVzRsFiltAttMap["directives"].(*schema.Set).List()
+
+									if oldVzRsFiltAttMap["filter_dn"] == newVzRsFiltAttMap["filter_dn"] {
+										found_same_filter_dn = true
+										if oldVzRsFiltAttMap["action"] != newVzRsFiltAttMap["action"] || oldVzRsFiltAttMap["priority_override"] != newVzRsFiltAttMap["priority_override"] || !reflect.DeepEqual(oldInTermFilterDirectives, newInTermFilterDirectives) {
+											if !(oldVzRsFiltAttMap["priority_override"] == "default" && newVzRsFiltAttMap["priority_override"] == "") {
```

Comment:
Could there be a case here where action is different but priority_override is default or empty and you ignore the change?